app/controllers/session_controller.rb

```diff
@@ -231,6 +232,49 @@ def create
     (user.active && user.email_confirmed?) ? login(user) : not_activated(user)
   end
 
+  def email_login_page
```

Comment:
I am going to do a rebase. Do you think we should merge this into `email_login` method?